pkg/jobs/jobspb/jobs.proto

```diff
@@ -153,15 +157,21 @@ message RestoreDetails {
     bytes info = 2;
   }
   repeated Tenant tenants = 13 [(gogoproto.nullable) = false];
+
   string override_db = 6 [(gogoproto.customname) = "OverrideDB"];
+
+  // The restore job has several atomic stages. For now, we keep track of which
+  // stages have completed via these flags. It would be nice if this was handled
```

Comment:
... I'm not sure what jobs would handle here? surely it shouldn't know about the states within a restore job?

Reply:
The comment may have been a bit too wishful. I was thinking that a job could be defined as a sequence of stages, each with their own corresponding rollback step if required. Haven't thought too deeply about it, but I'll update the comment. Better tracked elsewhere anyway.